tests/Mutator/Number/DecrementIntegerTest.php

```diff
@@ -303,6 +303,39 @@ public function provideMutationCases(): array
 PHP
                 ,
             ],
+            'It decrements an assignment' => [
+                <<<'PHP'
+<?php
+
+$foo = 10;
+PHP
+                ,
+                <<<'PHP'
+<?php
+
+$foo = 9;
+PHP
+            ],
+            'It decrements an assignment of 0' => [
+                <<<'PHP'
+<?php
+
+$foo = 0;
+PHP
+                ,
+                <<<'PHP'
+<?php
+
+$foo = -1;
+PHP
+            ],
+            'It does not decrement an assignment of 1' => [
```

Comment:
could you please remind why don't we decrement `1`?

Reply:
Because of the `OneZeroInteger` mutator, which does exactly that.

Reply:
ah, thanks, probably worths adding a comment before `if (!$node instanceof Node\Scalar\LNumber || $node->value === 1) {`